src/services/blockchain/ethereum/ethereum-blockchain-service.ts

```diff
@@ -41,42 +43,83 @@ export default class EthereumBlockchainService implements BlockchainService {
     }
```

Comment:
At some point we'd likely want to not just use the default provider and explicitly use Infura or similar.

Reply:
err, see the lines just above this.

Reply:
👍